src/components/OntologyBrowser/index.tsx

```diff
@@ -26,7 +26,7 @@ type ModalState = {
   error?: Error | null;
 };
 
-const { Title } = Typography;
+const ontologyRegex = new RegExp('([A-Za-z_]+).name');
```

Comment:
Are we sure it filters only and exactly what is needed? it seems to accept a lot of possibilities